docs/user/ppl/cmd/timechart.rst

```diff
@@ -132,109 +132,94 @@ PPL query::
 
     os> source=events | timechart span=1h count() by host
     fetched rows / total rows = 2/2
-    +---------------------+---------+-------+
-    | @timestamp          | host    | count |
-    |---------------------+---------+-------|
-    | 2023-01-01 10:00:00 | server1 | 4     |
-    | 2023-01-01 10:00:00 | server2 | 4     |
-    +---------------------+---------+-------+
-
-Example 2: Count events by minute with zero-filled results
+    +---------------------+---------+---------+
+    | @timestamp          | host    | count() |
+    |---------------------+---------+---------|
+    | 2023-01-01 10:00:00 | server1 | 4       |
+    | 2023-01-01 10:00:00 | server2 | 4       |
+    +---------------------+---------+---------+
+
+Example 2: Count events by minute
 ==========================================================
 
-This example counts events for each minute and groups them by host, showing zero values for time-host combinations with no data.
+This example counts events for each minute and groups them by host.
 
 PPL query::
 
     os> source=events | timechart span=1m count() by host
-    fetched rows / total rows = 16/16
-    +---------------------+---------+-------+
-    | @timestamp          | host    | count |
-    |---------------------+---------+-------|
-    | 2023-01-01 10:00:00 | server1 | 1     |
-    | 2023-01-01 10:00:00 | server2 | 0     |
-    | 2023-01-01 10:05:00 | server1 | 0     |
-    | 2023-01-01 10:05:00 | server2 | 1     |
-    | 2023-01-01 10:10:00 | server1 | 1     |
-    | 2023-01-01 10:10:00 | server2 | 0     |
-    | 2023-01-01 10:15:00 | server1 | 0     |
-    | 2023-01-01 10:15:00 | server2 | 1     |
-    | 2023-01-01 10:20:00 | server1 | 1     |
-    | 2023-01-01 10:20:00 | server2 | 0     |
-    | 2023-01-01 10:25:00 | server1 | 0     |
-    | 2023-01-01 10:25:00 | server2 | 1     |
-    | 2023-01-01 10:30:00 | server1 | 1     |
-    | 2023-01-01 10:30:00 | server2 | 0     |
-    | 2023-01-01 10:35:00 | server1 | 0     |
-    | 2023-01-01 10:35:00 | server2 | 1     |
-    +---------------------+---------+-------+
-
-Example 3: Calculate average CPU usage by minute
+    fetched rows / total rows = 8/8
+    +---------------------+---------+---------+
+    | @timestamp          | host    | count() |
+    |---------------------+---------+---------|
+    | 2023-01-01 10:00:00 | server1 | 1       |
+    | 2023-01-01 10:05:00 | server2 | 1       |
+    | 2023-01-01 10:10:00 | server1 | 1       |
+    | 2023-01-01 10:15:00 | server2 | 1       |
+    | 2023-01-01 10:20:00 | server1 | 1       |
+    | 2023-01-01 10:25:00 | server2 | 1       |
+    | 2023-01-01 10:30:00 | server1 | 1       |
+    | 2023-01-01 10:35:00 | server2 | 1       |
+    +---------------------+---------+---------+
+
```

Comment:
Why the results before and after for the same ppl query are different? please confirm the current result is correct comparing to SPL.